csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -5679,6 +5679,7 @@ Secondly, the program fulfills the following for all items of:
 
 * type `/$defs/version_t`: If any element doesn't match the semantic versioning, replace the all elements of type `/$defs/version_t` with the corresponding integer version. For that, CVRF CSAF converter sorts the items of `/document/tracking/revision_history` by `number` ascending according to the rules of CVRF. Then, it replaces the value of `number` with the index number in the array (starting with 1). The value of `/document/tracking/version` is replaced by value of `number` of the corresponding revision item. The match must be calculated by the original values used in the CVRF document. If this conversion was applied, for each Revision the original value of `cvrf:Number` MUST be set as `legacy_version` in the convertered document.
 * `/document/acknowledgments[]/organization` and `/vulnerabilities[]/acknowledgments[]/organization`: If more than one `cvrf:Organization` instance is given, the CVRF CSAF converter converts the first one into the `organization`. In addition, the converter outputs a warning that information might be lost during conversion of document or vulnerability acknowledgment.
+* `/document/lang`: If one or more CVRF element containing an `xml:lang` attribute exist and contain the exact same value, the CVRF CSAF converter converts this value into `lang`. If the values of `xml:lang` attributes are not equal, the CVRF CSAF converter outputs a warning that the language could not be determined and possibly a document with multiple languages was produced. In addition, it SHOULD also present all values of `xml:lang` attributes as a set.
```

Comment:
@tschmidtb51 Could you kindly help me understand the additional requirement? Maybe it is obvious, but where should the converter present all values of the found language attributes as a set? Thanks a lot in advance.

Reply:
In the warning.

Reply:
Hopefully clarified in [6b1d1f5](https://github.com/oasis-tcs/csaf/pull/428/commits/6b1d1f53b9f93bfebd179edfa5f363954bc7a5e2).